src/Avalonia.Base/Input/Raw/RawPointerEventArgs.cs

```diff
@@ -143,6 +143,14 @@ public record struct RawPointerPoint
         /// <inheritdoc cref="PointerPointProperties.YTilt" />
         public float YTilt { get; set; }
 
+        /// <inheritdoc cref="PointerPointProperties.ContactRect" />
+        public Rect ContactRect
+        {
+            get => _contactRect ?? new Rect(Position, new Size());
```

Comment:
it should be new Size(1,1) by default I believe

Reply:
@maxkatz6 Sorry, I do not think so. I think the size should be zero here as the WinUI do.